packages/foam-vscode/static/graphs/default/graph.js

```diff
@@ -300,6 +336,15 @@ try {
           Actions.selectNode(noteId);
         }
         break;
+      case "graphStyle":
+        const style = message.payload;
+        updateStyle(style);
```

Comment:
mmm... I think here you need a slightly different approach, to be in line with the other actions (and I think it will also simplify the code).

Because style is now part of the model (that is, it can be changed by the app), I would:
- add it to the model itself (and the style code in `initDataviz` needs to reference the model, not the `style` global)
- make `updateStyle` an action, and in its implementation make the changes to the style in the model 

If the design doesn't make sense to you let's chat.

Reply:
By model, do you mean this?
https://github.com/foambubble/foam/blob/32f40fa0decd20014a9e22699e39004a58bf01df/packages/foam-vscode/static/graphs/default/graph.js#L39-L49

Reply:
yes.
actions are used to update the model.
the model is rendered by the view (which was initialized once in `initGraph`, the `force-graph` library doesn't have a clear separation between model and view, but it's still good enough for us)

Reply:
New commits fix this